themes/default/content/blog/announcing-aws-native/index.md

```diff
@@ -33,6 +33,10 @@ The Pulumi AWS Native provider can be used in combination with the classic Pulum
 
 In this example, we can see how the new AWS S3 Object Lambda feature can be used via the AWS Native provider, with access to the full API defined by the S3 team at AWS, and combined naturally with another resource managed by the classic Pulumi AWS provider:
 
+{{< chooser language "typescript,python,csharp,go" >}}
+
+{{% choosable language typescript %}}
+
 ```typescript
 const bucket = new awsclassic.s3.Bucket("source");
```

Comment:
Should we update this to use awsnative as well given we're doing that in the other examples?

Reply:
```suggestion
const bucket = new awsnative.s3.Bucket("source");
```